docs: data cleanup / update structure (#453)
regen-network/regen-ledger: docs/regen-ledger/README.md

- Exposes an API server with both gRPC and REST interfaces for querying blockchain state and sending transactions
- Exposes a command line interface for account creation and key management as well as for querying blockchain state and sending transactions

Regen Ledger has two custom modules in development that support our desired functionality for ecological data, claims and credits.

### Ecocredit Module

::: tip COMING SOON

> An experimental version of the ecocredit module is available on [Regen Devnet](/getting-started/live-networks.html#regen-devnet). Regen Ledger v2 will include the first official version of the ecocredit module.

Reviewer: ```suggestion
An experimental version of the ecocredit module is available on [Regen Devnet](/getting-started/live-networks.html#regen-devnet). Regen Ledger v2 will include the first official release of the ecocredit module, making the ecocredit module available on Mainnet.
```

:::

The **Ecocredit Module** is our module for managing the issuance, trading, and retiring of credits pertaining to verifiable changes in ecosystem health.

> Initially, this module will be used for Regen Network's own [CarbonPlus Grasslands](https://regen-registry.s3.amazonaws.com/GHG+and+Co-Benefits+in+Grazing+Systems+Credit+Class.pdf) credit, but we've designed this module first and foremost to be an open platform for all credit designers, something like an ERC20 token standard for ecosystem service credits.

Author: You're right. It is more like an NFT. I think people are more familiar with ERC20, which is why it was originally written this way but I think it also makes sense to reference ERC720, especially if we provide a link.

Check out [Ecocredit Module Overview](./modules/ecocredit/) for more information about the ecocredit module.

### Data Module

::: tip COMING SOON

> An experimental version of the data module will be available soon. The first official version of the data module will be included in Regen Ledger v3.

Reviewer: ```suggestion
An beta version of the data module will be available in experimental builds of Regen Ledger in Q4 2020, and we are targeting a first release on Mainnet in Regen Ledger v3.
```

:::

High quality and verifiable ecological data is an essential component in any modern ecosystem service marketplace. The **Data Module** is intended to sit alongside the Ecocredit Module, serving as a generic repository for more complex metadata pertaining a credit batch or ecosystem service project.

The basic functionality of the data module includes storing, timestamping, and digitally signing data on Regen Ledger.

Check out [Data Module Overview](./modules/data/) for more information about the data module.

## Additional Functionality

Supplemental to the core featureset described, Regen Ledger has out-of-the-box support for the creation of DAOs, multi-sig wallets, and smart contracting capabilities. These more complex features are enabled through an additional set of modules from the Cosmos ecosystem:

- **Groups Module** – allowing nested accounts, or subkeys, with custom voting schemas for message execution
- **CosmWasm Integration** – enabling WASM based smart contracts to live on Regen Ledger

### Groups Module

::: tip COMING SOON

> The first official version of the group module will be included in the next release of Cosmos SDK (v0.44) and then included in Regen Ledger v3.

Reviewer: ```suggestion
The first official version of the group module will be included in the next release of Cosmos SDK (v0.44) and then included in Regen Ledger v3 for release on Mainnet.
```
